pkgs/development/python-modules/pywikibot/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  mwparserfromhell,
+  requests,
+  packaging,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "pywikibot";
+  version = "9.3.1";
```

Comment:
```suggestion
  version = "9.4.1";
```

Reply:
Updated to latest release, which is now 9.5.0.